devops/actions/run-tests/cts/action.yml

```diff
@@ -67,7 +67,7 @@ runs:
         # SYCL_CTS_EXCLUDE_TEST_CATEGORIES - Optional file specifying a list
         # of test categories to be excluded from the build.
         echo "::group::Excluded test categories"
-        cat $cts_exclude_filter
+        [ -f "$cts_exclude_filter" ] && cat "$cts_exclude_filter"
```

Comment:
lgtm but when could it not exist?

Reply:
for example when we test old branches:) - fresh example https://github.com/intel/llvm/actions/runs/14133470110/job/39601252637

Reply:
![image](https://github.com/user-attachments/assets/2dade9b0-0002-4793-af49-ab38e96bde8c)

Reply:
ah ok